jpsreport/methods/Method_I.cpp

```diff
@@ -150,9 +151,9 @@ bool Method_I::Process(
         } else {
             if(IsPointsOnOneLine(XInFrame, YInFrame)) {
                 if(fabs(XInFrame[1] - XInFrame[0]) < dmin) {
-                    XInFrame[1] += offset;
+                    XInFrame[1] += joffset;
```

Comment:
Why is it joffset in method I? What is the J standing for?

Reply:
**J**uPedSim

Reply:
renamed macro to JPS_OFFSET and moved it to Macros.h